resteasy-core/src/main/java/org/jboss/resteasy/core/ResteasyProviderFactoryImpl.java

```diff
@@ -1249,6 +1256,20 @@ public String toString(Object object, Class clazz, Type genericType, Annotation[
       }
       return object.toString();
    }
+   
+   public String toString() {

```

Comment:
Hi ron,

I don't get the point of this `toString()` impl. 
If I understand well, in addition to the java standard `toString()` representation it will list all `ParamConverterProvider` registered right ? What for ?

Reply:
Hmmm, it must be something I put in for testing and forgot to remove. Good catch.